estimatey-client/src/utils/getInitials.tsx

```diff
@@ -0,0 +1,15 @@
+export const getInitials = (displayName?: string): string => {
+
+    try {
+        if (!displayName) return "--";
+
+        const initialsArray = displayName?.split(" ").map(word => word ? word[0] : "");
```

Comment:
I know you love a good nit pick, but can just be `displayName.split ...` instead of `displayName?.split ...` as already asserted that it's not falsey :)